assignment/week03/travel-recommendation/src/components/NonRandom/NonRandom.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import { BtnBox, MainBox,Description,AnswerBox, Btn, LevelBox } from '../../styles/Design';
+import { questionList, temporaryAnswer } from './QuestionList';
+
+const NonRandom = ({id, setId, setIsStarted, calculator, setCalculator}) => {
+    const [nextBtnContent, setNextBtnContent]=useState('다음으로');
+    const [loaded,isLoaded]=useState(false);
+
+    useEffect(()=>{
+        id>=2 ? setNextBtnContent("결과보기") : setNextBtnContent("다음으로");
+        isLoaded(true);
+    },[id]);
+
+    const onAnswerClick=(idx)=>{
+        const newTemporary = temporaryAnswer[id].map((each,i)=>idx===i ? true : false);
+        temporaryAnswer[id]=newTemporary;
+        calculator[id]=idx;
+        setCalculator([...calculator]);
+    }
+
+    const onNextClick=()=>{
+        setId(id+1);
+    }
+    const onBackClick=()=>{
+        id-1>=0
+            ? setId(id-1) 
+            : (setIsStarted(false), setId(0))       
+    }
+    return (
+        <>
+            <Description>{questionList[id].question}</Description>
+            <LevelBox>
+                {id+1} / 3
```

Comment:
여기서 '3'도 상수로 바꿔주는건 어떨까용 ?!
지금은 3단계로 이루어져 있지만 나중에 단계가 추가되거나, 제거될수도 있는데 그때마다 숫자 값을 찾아서 변경하기엔 번거로울수 있습니댱
그래서 저는 TOTAL_STAGE 라는 상수를 하나 선언해놓고 
```suggestion
                {id+1} / {TOTAL_STAGE}
```
이런식으로 활용했어요!!

사소하지만 magic number 하드코딩은 지양하는 습관을 들이는게 좋다구 생각합니당!